packages/xrpl-crypto/HISTORY.md

```diff
@@ -0,0 +1 @@
+# @xrpl/crypto Release History
```

Comment:
Generally we update the HISTORY.md as we add things, so this should include the initial set of changes for the new package I believe.